src/test/regress/pg_regress_multi.pl

```diff
@@ -354,6 +356,11 @@ sub revert_replace_postgres
   push(@pgOptions, '-c', "wal_level=replica");
 }
 
+if ($maxPoolSize == 0)
+{
+  push(@pgOptions, '-c', "citus.max_pool_size=0");
```

Comment:
ops, I've changed the GUC name but not changed here. I realized from the code-cov tool, so I'll fix it. So, the code-coverage is pretty important tool :)